packages/enzyme-test-suite/test/ReactWrapper-spec.jsx

```diff
@@ -3223,6 +3223,133 @@ describeWithDOM('mount', () => {
       expect(wrapper.isEmptyRender()).to.equal(data.expectResponse);
     });
 
+    describe('nested nodes', () => {
+      class RenderChildren extends React.Component {
+        render() {
+          return this.props.children;
+        }
+      }
+
+      class RenderNull extends React.Component {
+        render() {
+          return null;
+        }
+      }
+
+      describeIf(is('< 16'), 'rendered nodes', () => {
```

Comment:
these tests should still pass on 16+, no? we can run them unconditionally?
```suggestion
      describe('rendered nodes', () => {
```